sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -529,7 +523,7 @@ export interface TopicRuntimePropertiesResponse extends TopicRuntimeProperties,
 
 // @public
 export interface WaitTimeOptions {
-    maxWaitTimeInMs: number;
+    maxWaitTimeInMs?: number;
```

Comment:
At this point `WaitTimeOptions` is used only in `ReceiveMessagesOptions` since we removed it from the options for the iterator. Can we inline this?

Reply:
Sure!

Reply:
Done!